pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -42,6 +44,7 @@ stdenv.mkDerivation rec {
     libxml2 # for xmllint for stripping GResources
     libxslt # for xsltproc for building docs
     gobject-introspection
+    (buildPackages.python3Packages.python.withPackages (ps: with ps; [ python3Packages.pygobject3 ])) # for cli tool
```

Comment:
I think this should go in buildInputs, since it's not used at compile time but at run time when you use the cli.

Reply:
Also I think you need to take off buildPackages, just use python3Packages alone.

Reply:
I looked at how systemd does it (and did it the same way here). 
You can compare if you want to.
Also while not strictly bein necessary for compilation meson fails the build without python because it replaces the shebang with the absolute path.

If you still think you would prefer the changes you suggested let me know and I'll change them.